### DISCUSSION TITLE
Starter Site

### DISCUSSION THREAD

**Discussion by MosheGramovsky** [bootscore/bootscore#524] [Q&A]
Do you sell bootScore as a starter site? Ideally something with the landing page, shop, and some of the documentation pages?

**Comment by crftwrk** [comment]
Hi Moshe,

yes, this is something we have wanted to do for years, but we haven't had the time to prepare them yet. Hopefully, we will have some initial starter themes ready by the beginning of 2024. However, here is a quick preview of what to expect. Maybe this will give you a little inspiration.

- https://themes.bootscore.me/bc-fashion/
- https://themes.bootscore.me/bs-corporate/
- https://themes.bootscore.me/bs-portfolio/

**Comment by MosheGramovsky** [comment]
Thank you for the reply. We would definitely make a purchase of a starter site, plus the add-ons, such as for light mode/dark mode. Your documentation is amazing too. We would also purchase a theme/template/starter site for the documentation as well (if you made one).
